### ISSUE TITLE
Take Over mode cannot resolve types in nested project

### ISSUE THREAD

**Issue by sqal** [johnsoncodehk/volar#1341]
**Version:**
0.34.16

**Reproduction link:**

https://github.com/sqal/volar-types-resolve-bug

**Steps to reproduce:**
- Clone & install deps `npm install`
- Open packages/app/src/index.ts - types for gtag are correctly resolved
- Enable Take Over mode (disable built-in TypeScript and JavaScript Language Features)
- Now TS cannot resolve @types from root node_modules `Cannot find name 'gtag'.ts(2304)`. For some reason I have to add typeRoots to my tsconfig.

```json
"typeRoots": [
  "../../node_modules/@types",
  "./node_modules/@types"
]
```

Any idea why resolving @types for nested projects does not work as expected when Take Over mode is enabled? I would be grateful if someone could tell me whether I have done something wrong here or this is indeed a bug.  Thank you.

**Comment by johnsoncodehk** [comment]
It seems volar's bug but can't found it out yet.

Maybe related:

- https://github.com/johnsoncodehk/volar/blob/841562c1bb423f9d2bd80a2aed92876b37a7a75d/packages/vue-language-server/src/project.ts#L181-L183
- https://github.com/microsoft/TypeScript/blob/0d9a9dc75d12afe4feb8cfeb12a29f1a87c1360e/src/compiler/types.ts#L8450-L8453

**Comment by johnsoncodehk** [comment]
Fixed in v0.38.7.
